docs/update_release_notes.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Script to help with the SMQTK release process. Performs the following steps:
+#   - Poetry version (major, minor, or patch)
+#   - Rename release_notes/pending_release file to release_notes/version
+#   - Add reference to new release notes file in release_notes.rst
+#   - Add new release notes stub file
+#
+# Two git commits are created. One for the version bump and one for the new
+# release notes stub file.
+#
+
+# Check args
+if [ $# != 1 ]
+then
+  echo "Please enter valid version bump type. Options: major, minor, or patch"
+  exit 1
+fi
+
+if [ $1 != 'major' ] && [ $1 != 'minor' ] && [ $1 != 'patch' ]
```

Comment:
Variable expansion should be double-quoted. There are various other locations in this script where variable expansion should be wrapped in quotes. See the comment on using shellcheck.